boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -376,20 +380,43 @@ TEST_F(DimStateMachineTest, imd_led_control_in_drive_state)
 // DIM-6
 TEST_F(DimStateMachineTest, bspd_led_control_in_drive_state)
 {
-    App_CanRx_FSM_ERRORS_SetSignal_BSPD_FAULT(can_rx_interface, false);
+    App_CanRx_FSM_NON_CRITICAL_ERRORS_SetSignal_BSPD_FAULT(
+        can_rx_interface, false);
     App_SharedStateMachine_Tick100Hz(state_machine);
     ASSERT_EQ(0, turn_on_bspd_led_fake.call_count);
     ASSERT_EQ(1, turn_off_bspd_led_fake.call_count);
 
-    App_CanRx_FSM_ERRORS_SetSignal_BSPD_FAULT(can_rx_interface, true);
+    App_CanRx_FSM_NON_CRITICAL_ERRORS_SetSignal_BSPD_FAULT(
+        can_rx_interface, true);
     App_SharedStateMachine_Tick100Hz(state_machine);
     ASSERT_EQ(1, turn_on_bspd_led_fake.call_count);
     ASSERT_EQ(1, turn_off_bspd_led_fake.call_count);
 
-    App_CanRx_FSM_ERRORS_SetSignal_BSPD_FAULT(can_rx_interface, false);
+    App_CanRx_FSM_NON_CRITICAL_ERRORS_SetSignal_BSPD_FAULT(
+        can_rx_interface, false);
     App_SharedStateMachine_Tick100Hz(state_machine);
     ASSERT_EQ(1, turn_on_bspd_led_fake.call_count);
     ASSERT_EQ(2, turn_off_bspd_led_fake.call_count);
 }
 
+TEST_F(DimStateMachineTest, error_on_7_segment_displays)
+{
+    App_ErrorTable_SetBmsStackWaterMarkAboveThresholdTask1Hz(error_table);
```

Comment:
I was hoping to replace this line with `App_CanRx_SettBmsStackWaterMarkAboveThresholdTask1Hz`, but since Error Table and Can RX are separate right now I can't do that. Not sure if that's a big deal.